torch/csrc/jit/script/init.cpp

```diff
@@ -246,9 +246,25 @@ struct ModuleValue : public SugaredValue {
     }
     throw ErrorReport(loc) << "module has no attribute '" << field << "'";
   }
+
   // call module.forward
   virtual std::shared_ptr<SugaredValue> call(SourceRange loc, Method & caller, at::ArrayRef<NamedValue> inputs, at::ArrayRef<NamedValue> attributes, size_t n_binders) override {
-    return attr(loc, caller, "forward")->call(loc, caller, inputs, attributes, n_binders);
+    py::object py_module = py::cast(module);
+    if(!py::isinstance(py_module, py::module::import("torch.jit").attr("_ConstModuleList")))
+      return attr(loc, caller, "forward")->call(loc, caller, inputs, attributes, n_binders);
+    std::shared_ptr<SugaredValue> result;
```

Comment:
I think the input for the first module is also required to have exactly one input. Can you add a check here?